apps/admin-gui/src/app/admin/admin.module.ts

```diff
@@ -30,6 +30,8 @@ import { UiAlertsModule } from '@perun-web-apps/ui/alerts';
 import { AdminExtSourcesComponent } from './pages/admin-page/admin-ext-sources/admin-ext-sources.component';
 import { EditAttributeDefinitionDialogComponent } from '../shared/components/dialogs/edit-attribute-definition-dialog/edit-attribute-definition-dialog.component';
 import { EntitylessAttributeKeysListComponent } from '../shared/components/entityless-attribute-keys-list/entityless-attribute-keys-list.component';
+// tslint:disable-next-line:nx-enforce-module-boundaries
+import { PerunSharedComponentsModule } from '../../../../../libs/perun/components/src/lib/perun-components.module';
```

Comment:
```suggestion
import { PerunSharedComponentsModule } from ''@perun-web-apps/perun/components';
```
All imports from libraries should start with `@libname`. To make this work, you also need to edit the `index.ts` file located inside the new created library and export the module: 
```
export * from './lib/perun-components.module';
```

Reply:
If your IDE shows you an error, try to remove invalidate and restart caches or remove `node_modules` and reinstall them.